src/hooks.client.ts

```diff
@@ -0,0 +1,21 @@
+import { handleErrorWithSentry, replayIntegration } from "@sentry/sveltekit";
+import * as Sentry from '@sentry/sveltekit';
+
+Sentry.init({
+  dsn: 'https://27d41dc8bb67b596f137924ab8599e59@o1063647.ingest.us.sentry.io/4507497727000576',
```

Comment:
Due to this, both Stage and Production will go to same project. We might want another ENV variable for DSN

Reply:
should be fine, we can make it dependend on the url 👍🏻